kettle/stream.py

```diff
@@ -41,23 +42,30 @@
 import make_db
 import make_json
 
+
 MAX_ROW_UPLOAD = 10 # See https://github.com/googleapis/google-cloud-go/issues/2855
 
-def process_changes(results):
+
+def should_exclude(objectID, bucketID, buckets):
+    job_name = objectID.rsplit('/')[-3] # See tests for refernce
```

Comment:
i think a comment of a sample objectID would also go well here [-3] looks like magic :)

Reply:
done